hack/run.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash 
+
+set -e
+
+ROOT_DIR=$(dirname $0)/..
+
+action="$1"
+
+function usage() {
+	cmd="$1"
+
+	echo ""
+	echo "Usage                                                for more information read DEVELOPMENT.md"
+	echo "$cmd <command>"
+	echo ""
+	echo "command:"
+	echo "   install                                           Install Knative components for local dev"
+	echo "   e2e-debug <test-name> <test-dir>                  Quick debug"
+	echo "   update-cert-manager                               Update Cert manager"
+	echo "   generate-yamls <repo-root-dir> <output-file>      Generate all repo yamls "
+	echo "   update-reference-docs                             Update reference docs"
+	echo "   release                                           Release new version"
+	echo "   teardown                                          Teardown installed components for local dev"
+	echo "   update-checksums                                  Update checksums"
+	echo "   update-codegen                                    Update codegen"
+	echo "   update-deps                                       Update deps"
+	echo "   verify-codegen                                    Verify codegen"
+	echo ""
+}
+
+if [[ "${action}" == "install" ]]; then
+	source "${ROOT_DIR}"/hack/install.sh 
+elif [[ "${action}" == "update-cert-manager" ]]; then
+	source "${ROOT_DIR}"/hack/update-cert-manager.sh "$2" "$3"
+elif [[ "${action}" == "e2e-debug" ]]; then
+	source "${ROOT_DIR}"/hack/e2e-debug.sh "$2" "$3"
+elif [[ "${action}" == "generate-yamls" ]]; then
+	source "${ROOT_DIR}"/hack/generate-yamls.sh "$2" "$3"
+elif [[ "${action}" == "update-reference-docs" ]]; then
+	source "${ROOT_DIR}"/hack/update-reference-docs.sh
+elif [[ "${action}" == "release" ]]; then
```

Comment:
It seems like release option doesn't work as expected on my end. The output is different from manually run release.sh. Could you investigate on what would the problem be?

Reply:
@Leo6Leo I don't think we need a release option, I can't think of a scenario where someone would be releasing knative locally